### ISSUE TITLE
How to change multiple wheat varieties in the same long-term simulation

### ISSUE THREAD

**Issue by PanxinZhang** [APSIMInitiative/APSIMClassic#2161]
If it can only be achieved by writing code, I hope you can give me an example for me to learn.

**Comment by peter-devoil** [comment]
Yes, manager code is very powerful and can do just as you describe. There is already some logic in the planting rule that switches cultivars depending on date, it's not hard to extend that to lists or tables.

**Comment by PanxinZhang** [comment]
Thanks for answer.I don't have the basis of the relevant code before, can you provide a part of the management code about the variety replacement for me to learn?

> Yes, manager code is very powerful and can do just as you describe. There is already some logic in the planting rule that switches cultivars depending on date, it's not hard to extend that to lists or tables.

**Comment by peter-devoil** [comment]
Here's the implementation from earlier (from the "new simulation" button: ![image](https://user-images.githubusercontent.com/1924386/174517623-ef953516-763f-4ea5-b785-b29005e3b453.png)

There's a partiually written intro to using apsim managers [here](https://github.com/APSIMInitiative/APSIMClassic/files/5570167/How.to.write.manager.scripts.pptx).
